frodo.js

```diff
@@ -4,131 +4,132 @@
  * This plugin performs initialization (like calling mermaid.initialize())
```

Comment:
Nice job on the comprehensive test coverage, it gives a lot of confidence in the stability of the new feature.